receiver/skywalkingreceiver/README.md

```diff
@@ -0,0 +1,24 @@
+# Skywalking Receiver
+
+Receives trace data in [Skywalking](https://skywalking.apache.org/) format.
+
+Supported pipeline types: traces
+
+## Getting Started
```

Comment:
I'm missing a section with the list of protocols, their defaults and which one is served automatically or not, e.g. [like for Jaeger](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jaegerreceiver#getting-started)